### ISSUE TITLE
FA game path cancel/close exception

### ISSUE THREAD

**Issue by kyx0r** [FAForever/client#933]
When the client asks user for FA path, if to close or cancel the window a nasty exception shows up. 
![123](https://user-images.githubusercontent.com/33268383/34331199-a1a34f60-e8f2-11e7-826a-78b2d60e0cb4.png)
Exception:
![1234](https://user-images.githubusercontent.com/33268383/34331238-f03aed54-e8f2-11e7-9eed-d125ad75e780.PNG)
 
I have made some investigation and if to change the line 571 in fa\updater.py (to pass) it will solve this; but in exchange of killing the exemption handler. Also, the code for most of this Qvbox is located in wizards.py however, the code for close and cancel buttons is not present (maybe other file? .ui is not telling a lot either). It could also be server problem.

In conclusion, this is not a too big of an issue, but i found it somewhat necessary to mention.

**Comment by Wesmania** [comment]
It seems to me that not setting the path causes it to stay as None, and the updater happily runs until somewhere he has to concatenate some paths and he encounters a None. Of course, since it follows a brilliant path of catching an exception, continuing, then rethrowing it, we have no idea where it's happening :/

Getting rid of the except block in line 550 should reveal the exact line where the issue is.
